color/src/main/java/com/afollestad/materialdialogs/color/DialogColorChooserExt.kt

```diff
@@ -26,7 +33,10 @@ typealias ColorCallback = ((dialog: MaterialDialog, color: Int) -> Unit)?
  * @param initialSelection The optionally initially selected color literal integer.
  * @param waitForPositiveButton When true, the selection isn't invoked until the user selects
  *    a color and taps on the positive action button. Defaults to true if the dialog has buttons.
- * @param selection An optional callback invoked when the user selects a color.
+ * @param allowCustomColor allows to select a color with an (A)RGB slider view -
+ *     ATTENTION: the neutral button will be set to switch between presets and color selector + autoDismiss is disabled!
```

Comment:
The neutral button is deprecated - not just by me but also by Google

Reply:
I don't believe the cancel button is 😄 

Hmmm, a ViewPager could be a cool solution